src/client/dfs/dfs.c

```diff
@@ -2576,7 +2576,7 @@ dfs_obj_global2local(dfs_t *dfs, int flags, d_iov_t glob, dfs_obj_t **_obj)
 	/** Check container uuid mismatch */
 	rc = dc_cont_hdl2uuid(dfs->coh, &coh_uuid, &cont_uuid);
 	if (rc != 0)
-		D_GOTO(out, rc);
+		D_GOTO(out, daos_der2errno(rc));
```

Comment:
Shouldn't this be "D_GOTO(out, rc = daos_der2errno(rc));" instead?

Reply:
Good catch, yes.